pyproject.toml

```diff
@@ -164,11 +164,32 @@ indent-width = 4
 # Allow unused variables when underscore-prefixed.
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 ignore = [
+  "E501",  # line too long
```

Comment:
This should be temporary, at least until we've done a refactoring pass for formatting. But it's a big refactor commit and not for today.